engine/apps/mobile_app/auth.py

```diff
@@ -42,4 +43,7 @@ def authenticate_credentials(self, token_string: str) -> Tuple[Optional[User], O
         except InvalidToken:
             return None, None
 
+        if auth_token.organization.is_moved:
+            raise OrganizationMovedException(auth_token.organization)
+
```

Comment:
We should handle the OrganizationDeleted at the same time like other auth classes
```
if auth_token.organization.deleted_at:
    raise OrganizationDeletedException(auth_token.organization)
```